.github/workflows/test-integration-vizro-core.yml

```diff
@@ -20,45 +20,27 @@ env:
 
 jobs:
   run:
-    name: Python ${{ matrix.python-version }} on ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
+    name: Python ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest]
         python-version: ["3.8", "3.9", "3.10", "3.11"]
 
     steps:
       - uses: actions/checkout@v4
-      - name: Get branch name
-        id: branch-name
-        uses: tj-actions/branch-names@v7
 
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
 
-      - uses: actions/cache@v3
-        with:
-          path: ${{ env.pythonLocation }}
-          key: ${{ matrix.os }}-${{ matrix.python-version }}-${{ steps.branch-name.outputs.current_branch }}-pip-${{ hashFiles('hatch.toml') }}-${{ hashFiles('pyproject.toml') }}
-          restore-keys: |
-            ${{ matrix.os }}-${{ matrix.python-version }}-${{ steps.branch-name.outputs.current_branch }}-pip-
+      - name: Install Hatch
```

Comment:
@l0uden I wasn't sure whether to remove this cache or not. Is it only useful on Windows or in general? We don't have it on our unit tests or any other github workflows so not sure whether it's actually speeding up anything or not? I feel like either it should be on all workflows or it should be on none of them?

Reply:
Cool, I'll leave it deleted then.

Can I also remove `.github/requirements.txt` now? I can't remember what that's needed for.

Reply:
Yes, it was used only for windows tests

Reply:
Perfect, I'll delete that too then.